utils/build/docker/java/install_drop_in.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eu
+
+if [ $(ls /binaries/opentelemetry-javaagent-r2dbc*.jar | wc -l) = 0 ]; then
+    LIBRARY_URL='https://repo1.maven.org/maven2/io/opentelemetry/javaagent/instrumentation/opentelemetry-javaagent-r2dbc-1.0/2.5.0-alpha/opentelemetry-javaagent-r2dbc-1.0-2.5.0-alpha.jar'
+    echo "install from Maven central: $LIBRARY_URL"
```

Comment:
Could you document in `docs/execute/binaries.md` what are the possible options ?

Reply:
Documented in 1cea91c12